common/src/main/java/tachyon/Constants.java

```diff
@@ -97,22 +97,23 @@
   public static final String LOGS_DIR = "tachyon.logs.dir";
   public static final String UNDERFS_ADDRESS = "tachyon.underfs.address";
   public static final String UNDERFS_HDFS_IMPL = "tachyon.underfs.hdfs.impl";
-  public static final String UNDERFS_HADOOP_CONFIGURATION = "tachyon.underfs.hadoop.configuration";
```

Comment:
actually they are not autogenerated---just Java IDE like intellij is smart enough to refactor property names meanwhile update the references to these properties in other files of the source code.